### PR TITLE
added alignment to text input

### PR DESCRIPTION
**Pull request by almeynman**

### CODE REVIEW

File: lib/inputs/stream_text_input.dart [hunk 1]
```diff
@@ -13,13 +13,15 @@ class StreamTextInput extends StatefulWidget {
...
   final bool obscureText;
+  final TextAlign align;
 
   const StreamTextInput({
     Key? key,
     required this.value$,
     this.onChange = defaultOnChange,
     this.hintText,
     this.keyboardType,
+    required this.align,
```
Comment (almeynman) [inline lib/inputs/stream_text_input.dart:24]: I think better to use default value, rather make this required. It would also make it backward-compatible

Comment (JustinMcKague) [reply lib/inputs/stream_text_input.dart]: updated

Comment (JustinMcKague) [reply lib/inputs/stream_text_input.dart]: updated

File: lib/inputs/stream_text_input.dart [hunk 2]
```diff
@@ -13,13 +13,15 @@ class StreamTextInput extends StatefulWidget {
...
   final bool obscureText;
+  final TextAlign? align;
 
   const StreamTextInput({
     Key? key,
     required this.value$,
     this.onChange = defaultOnChange,
     this.hintText,
     this.keyboardType,
+    this.align,
```
Comment (almeynman) [inline lib/inputs/stream_text_input.dart:24]: to set default value `this.align = TextAlign.center`

File: lib/inputs/stream_text_input.dart [hunk 3]
```diff
@@ -56,6 +58,7 @@ class _StreamTextInputState extends State<StreamTextInput> {
       builder: (context, snapshot) {
         return TextField(
           keyboardType: widget.keyboardType,
+          textAlign: TextAlign.center,
```
Comment (almeynman) [inline lib/inputs/stream_text_input.dart:61]: `textAlign: align`

Comment (JustinMcKague) [reply lib/inputs/stream_text_input.dart]: updated both lines

**Comment by almeynman** [comment]
Approved. Can't click Approve button, cause I opened PR

**Review by JustinMcKague — APPROVED** [review]

**Pull request merged**